File: java/seasar/autoRegist/src/test/DataImpl.java
package test;

public class DataImpl implements Data {

	private String name;
	private int point;

	public DataImpl() {
		this.name = "test";
		this.point = 100;
	}

	public DataImpl(String name, int point) {
		this.name = name;
		this.point = point;
	}

	public String getName() {
		return this.name;
	}

	public int getPoint() {
		return this.point;
	}

}

File: csharp/AspectDNG/wrapperTest/DataImpl.cs

using System;

public class DataImpl : IData
{
	public void CheckAudio(string bstrWMX, out int plPeakMin, out int plPeakMax)
	{
		plPeakMin = 10;
		plPeakMax = 100;

		Console.WriteLine("DataImpl.CheckAudio({0}, {1}, {2})", bstrWMX, plPeakMin, plPeakMax);
	}

	public void Print()
	{
		Console.WriteLine("DataImpl.Print()");
	}

	public string Test(string data, int i)
	{
		Console.WriteLine("DataImpl.Test({0}, {1})", data, i);
		return "DataImpl";
	}
}